keyboards/choc_taro/keymaps/via/rules.mk

```diff
@@ -0,0 +1,4 @@
+VIA_ENABLE = yes
+MOUSEKEY_ENABLE = no
+EXTRAKEY_ENABLE = no
+BOOTMAGIC_ENABLE = lite
```

Comment:
```suggestion
```
VIA supports mousekeys and media keys, so these should be left enabled if at all possible.